setup.py

```diff
@@ -15,7 +15,9 @@
 
 install_requires = [
     'Click>=6.0',
-    'cookiecutter>=1.6.0',
+    # We need the functionality provided by this branch. See this PR for more info:
+    # https://github.com/cookiecutter/cookiecutter/pull/907
+    'cookiecutter @ https://github.com/zillow/cookiecutter/tarball/strip_dir',
```

Comment:
do we actually want to fork cookiecutter? If so, should we just publish it with different name?

Reply:
I don't really want to do this but I couldn't work out a better quick fix solution to unblock us in the short term. I agree this isn't ideal but not sure of the best method to move forward as we need a feature that's been blocked at the PR level for 2.5 years upstream: https://github.com/cookiecutter/cookiecutter/pull/907

This seemed like the least effort for now to unblock us to hit our goals.

Reply:
We had a discussion offline about this and came to the conclusion that this is still the best short term option. Longer term options include including `cookiecutter` itself inside `battenberg` or lobbying to include the desired change upstream, either way this is outside of the scope of the current work and therefore we're keeping as is for now.